static/docs/user-guide/install/extra/code.md

```diff
@@ -0,0 +1,51 @@
+# Install from code
```

Comment:
let's just move the Development section as is for now. No need to move here parts of the Contributing Guide. They serve two very different purposes.

Reply:
I addressed this in 1774355e.